### PR TITLE
2.235.x

### PR DESCRIPTION
**Pull request by jglick**
Once 2.235.1 is released, could merge this, and could consider removing the 2.164.x line.

### CODE REVIEW

**Comment by jetersen** [comment]
Would be nice to land JCasC before 2.164 was removed

**Comment by jglick** [comment]
#225 you mean. Sure, this would be a few weeks away at a minimum.

**Comment by jglick** [comment]
```
java.lang.IllegalArgumentException: Last encoded character (before the paddings if any) is a valid base 64 alphabet but not a possible value. Expected the discarded bits to be zero.
	at org.apache.commons.codec.binary.Base64.validateCharacter(Base64.java:803)
	at org.apache.commons.codec.binary.Base64.decode(Base64.java:477)
	at org.apache.commons.codec.binary.BaseNCodec.decode(BaseNCodec.java:482)
	at org.apache.commons.codec.binary.BaseNCodec.decode(BaseNCodec.java:465)
	at org.apache.commons.codec.binary.Base64.decodeBase64(Base64.java:699)
	at com.cloudbees.plugins.credentials.SecretBytes.isSecretBytes(SecretBytes.java:302)
	at com.cloudbees.plugins.credentials.SecretBytesTest.isSecretBytes(SecretBytesTest.java:84)
```

**Comment by timja** [comment]
> ```
> java.lang.IllegalArgumentException: Last encoded character (before the paddings if any) is a valid base 64 alphabet but not a possible value. Expected the discarded bits to be zero.
> 	at org.apache.commons.codec.binary.Base64.validateCharacter(Base64.java:803)
> 	at org.apache.commons.codec.binary.Base64.decode(Base64.java:477)
> 	at org.apache.commons.codec.binary.BaseNCodec.decode(BaseNCodec.java:482)
> 	at org.apache.commons.codec.binary.BaseNCodec.decode(BaseNCodec.java:465)
> 	at org.apache.commons.codec.binary.Base64.decodeBase64(Base64.java:699)
> 	at com.cloudbees.plugins.credentials.SecretBytes.isSecretBytes(SecretBytes.java:302)
> 	at com.cloudbees.plugins.credentials.SecretBytesTest.isSecretBytes(SecretBytesTest.java:84)
> ```

We hit something similar at work it's likely this: https://issues.apache.org/jira/browse/CODEC-263

and normally caused by bad test data

**Comment by jglick** [comment]
Yeah this failure sounds vaguely familiar, just need to dig into it a bit.

**Comment by timja** [comment]
https://github.com/jenkinsci/jenkins/pull/4636#issuecomment-640883683

**Comment by jglick** [comment]
Funny, I had _just_ tracked the regression down to that PR. Working on a fix.

File: sample-plugin/pom.xml [hunk 1]
```diff
@@ -14,8 +14,8 @@
     <properties>
         <revision>11</revision>
         <changelist>-SNAPSHOT</changelist>
-        <bom>2.222.x</bom>
-        <jenkins.version>2.222.1</jenkins.version>
+        <bom>2.235.x</bom>
+        <jenkins.version>2.235</jenkins.version>
```
Comment (timja) [inline sample-plugin/pom.xml:18]: once released
```suggestion
        <jenkins.version>2.235.1</jenkins.version>
```

**Comment by jglick** [comment]
We _could_ remove 2.164.x at the same time. No strong feelings either way.

**Pull request merged**